### DISCUSSION TITLE
Deleting old, unused tags or making them unseen if weren't used in the chosen time window.

### DISCUSSION THREAD

**Discussion by constalozinschi** [langfuse/langfuse#3289] [Ideas]
### Describe the feature or potential improvement

Hi, i'm trying too make the most out of the dashboard as it currently is and i found as only solution in order to see stats relative to different LLMs, prompts etc., to insert them as tags on the traces. Having that done, and having assesed the most appropriate things i want to use tags for and the right names for them, i found myself having lots of old unused tags which make it hard to find and select the ones that i need. For the use i'm making now it would be great to make tags invisible if they havent been used in the current time window. Thanks.

**Comment by marcklingen** [comment]
Thanks for sharing! can you type the name of the tag to find the correct one?

**Reply by constalozinschi** [reply]
Thats a great insight thx 😄
